chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1467,7 +1525,10 @@ impl Actor for PeerManagerActor {
         // Start active peer stats querying.
         self.monitor_peer_stats_trigger(ctx);
 
+        // Read verified edges and broadcast them.
         self.broadcast_edges_trigger(ctx);
+
+        self.update_routing_table_trigger(ctx);
```

Comment:
comment: update routing table and prune edges that are no longer available.

Reply:
Thanks fixed. We are pruning edges that are no longer reachable. Those may be existing edges, but to which we can't find a valid path.

Reply:
```
        // Update routing table and prune edges that are no longer reachable.
        self.update_routing_table_trigger(ctx);
```